src/services/projectVerificationFormService.ts

```diff
@@ -153,6 +153,15 @@ export const updateProjectVerificationFormByUser = async (params: {
           projectVerificationId,
           isTermAndConditionsAccepted,
         });
+
+      break;
+    }
+    case PROJECT_VERIFICATION_STEPS.SUBMIT: {
+      updatedProjectVerificationForm =
+        await updateTermsAndConditionsOfProjectVerification({
+          projectVerificationId,
+          isTermAndConditionsAccepted: true,
```

Comment:
I think you shouldn't change isTermAndConditionsAccepted state on SUBMIT step. If user has not accepted Term And Conditions, we shouldn't change that to true when PROJECT_VERIFICATION_STEPS.SUBMIT is called.

Reply:
He can't submit last step if all steps aren't checked, but can I somehow make some update here because I need `updatedProjectVerificationForm` object for code below?

Reply:
let me try

Reply:
@RamRamez when you get time please check, thx ;)

Reply:
Thanks @kkatusic. Approved!

I think there's a typo in `getdProjectVerificationData`